### ISSUE TITLE
90-degree rotation is inconsistent

### ISSUE THREAD

**Issue by urdh** [mrkkrp/JuicyPixels-extra#15]
It seems that the 90-degree rotation functions are inconsistent; given the documentation I would expect `rotateLeft90` and `rotate180 . rotateRight90` to do the same thing (and `rotateLeft90 . rotateRight90` to be the identity function), but this doesn't seem to be the case.

Should `rotateLeft90` not have the following definition?

```haskell
rotateLeft90 :: Pixel a => Image a -> Image a
rotateLeft90 img@Image {..} =
  generateImage gen imageHeight imageWidth
  where
    gen x y = pixelAt img (imageWidth - 1 - y) x
{-# INLINEABLE rotateLeft90 #-}
```

**Comment by mrkkrp** [comment]
It looks like you're correct. I'm going to push a fix.

**Comment by mrkkrp** [comment]
I fixed this in #16, and published version 0.4.1 on Hackage. Thanks for reporting it.
